config.go

```diff
@@ -395,6 +397,17 @@ func loadAndValidateConfig(interceptor signal.Interceptor) (*Config, error) {
 		cfg.Lnd.RPCMiddleware.Enable = true
 	}
 
+	// For the integration of tapd with lnd, we need to allow tapd to send
+	// custom error messages to peers through the SendCustomMessage RPC in
+	// lnd. Since the error messages aren't in the custom range, we
+	// explicitly need to allow them.
+	if !cfg.lndRemote {
```

Comment:
ok so for remote mode i assume (like rpc middle ware stuff) that we will just start up but then quickly tap will notice it fails to deliver custom msgs to lnd?